CHANGELOG.md

```diff
@@ -1,7 +1,25 @@
 # Version history
 
 <!-- SPDX-License-Identifier: CC0-1.0 -->
-<!-- written in 2019 - 2022 by The Foundation for Public Code <info@publiccode.net> -->
+<!-- SPDX-FileCopyrightText: 2019-2022 The Foundation for Public Code <info@publiccode.net>, https://standard.publiccode.net/AUTHORS -->
+
+## Version 0.4.1
+
+December 5th 2022: 🖳 the twelth draft clarifies document maturity.
+
+* Document maturity focuses on whether or not versions of the codebase are ready to use.
```

Comment:
In the past, we've used a mix of past and present tense in these bullets. I think it would read more cleanly if they were all in the past - after all, this is a list of changes we've already made.
This doesn't need to block publication today.